web/src/components/MultiFormPage/MultiFormPage.jsx

```diff
@@ -7,12 +7,13 @@ import TransferableExperience from '../TransferableExperience/TransferableExperi
 import Education from '../Education/Education.jsx';
 import Project from '../Project/Project.jsx';
 import ProfileVsJob from '../ProfileVsJob/ProfileVsJob.jsx';
-import IconSlide from '../IconSlide/IconSlide.jsx';
 import Button from '../Button/Button.jsx';
-import ErrorState from '../ErrorState/ErrorState.jsx';
```

Comment:
This change should not happen as it just adds noise can you revert it so that we only make meaningful changes in the code